CS328-Notes/notebooks/2022_02_10_Expansion_Conductance_Graphs_Sparset_Cut.ipynb

```diff
@@ -0,0 +1,379 @@
+    " $$\n",
+    "\n",
+    " \n",
+    " * Where  n==D(V) , total number of vertices of the graph .\n",
+    "\n",
+    " * This result is also produce cut similar to Min_cut .\n",
+    "\n",
+    "                \n",
+    " ### 3)Product:\n",
+    "\n",
+    " $$               \n",
+    "{Expansion(S)=\\frac{e(S,\\overline{S})}{{|S|}{|\\overline{S}|}}}\n",
+    " $$\n",
+    "\n",
+    "* This term gives similar result to putting minimum in the denominator.\n",
+    "\n",
+    "```{admonition} Note\n",
+    "This above definition was proposed by Computer vision researcher *Jitendra Malik* and his team.\n",
+    "```\n",
+    "    \n",
+    "\n",
+    " ***\n",
+    "\n",
+    "\n",
+    "## (2) Conductance\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "* It is an indicator of Surface/volume property of graph and we use degree instead of cardinality in the $Expansion(S,\\overline {S})$ as it gives efficient contibution term to the volume aspect of the graph.\n",
+    "\n",
+    "$$ \n",
+    "{Conductance(S)=\\frac{e(S,\\overline{S})}{min_({d(S)},{d(\\overline{S})})}}\n",
+    "$$  \n",
+    "\n",
+    "\n",
+    "Let the subset be of the graph G(V,E) be S and $\\overline{S}$ be the complement of the of subset S.\n",
+    "\n",
+    "* d(S)= Sum of all degree of vertices in Subset S\n",
+    "* $d(\\overline{S})$=Sum of all degree of vertices in Subset $ \\overline{S}$\n",
+    "\n",
+    "\n",
+    "\n",
+    "```{figure} ../assets/2022_02_10_Expansion_conductance_graph_sparsecut/Dense_subgraph.jpeg\n",
+    ":name: figure-2\n",
+    "\n",
+    "conductance on Graph(G)\n",
+    "```\n",
+    "\n",
+    "Here S=bhcde and $\\overline{S}$=agf\n",
+    "\n",
+    "\n",
+    " \n",
+    " * ${e(S,\\overline{s})} $=Number of edges in between S and $\\overline{S}=\\sum_{a\\in S,b\\in \\overline{s}}ab $ \n",
+    "    \n",
+    " \n",
+    " * ${d(S)}$ =$\\sum_{u\\in S}d(u)$  =Degree of the vertices in the subset\n",
+    "    \n",
+    "  $$ Conductance(G)=Min(Conductance(S\\subset G(V,E))) $$\n",
+    "\n",
+    "In the above example Graph G:\n",
+    " \n",
+    "Here S=bhcde and $\\overline{S}$=agf\n",
+    "    \n",
+    "    \n",
+    " * **Note:** For the Regular graph,\n",
+    "    \n",
+    "$$Conductance(S)=\\frac{e(S,\\overline{S})}{min_({d(S)},{d(\\overline{S})})}=\\frac{1}{d}\\frac{e(S,\\overline{S})}{min_({|S|},{|\\overline{S}|})}=\\frac{1}{d}(Expansion(S))\n",
+    "$$\n",
+    "    \n",
+    " * For the irregular graphs,conductance term is very crucial as degree caputures the importance|volume property of the graph.  \n",
+    "\n",
+    "\n",
+    "## Sparsest Cut\n",
+    "\n",
+    "For a graph G ,Cut resulting the subset with minimum Conductance$(S,\\overline{S})$\n",
+    "\n",
+    "* It is also the conductance of Graph G(V,E).\n",
+    "\n",
+    " $$ Conductance(G)=Min(Conductance(S\\subset G(V,E))) $$\n",
+    "\n",
+    "```{figure} ../assets/2022_02_10_Expansion_conductance_graph_sparsecut/sparse_cut.jpeg\n",
```

Comment:
`../assets/2022_02_10_Expansion_conductance_graph_sparsecut/sparse_cut.jpeg\n` needs to be in next line after ```{figure}

Reply:
I also looked into other codes about this image loading issue, but I haven't seen any difference. They used ../assets ... in the same line after ```{figure }.